src/components/forms/controls/Checkbox/Checkbox.tsx

```diff
@@ -13,19 +13,35 @@ export { cl as CheckboxClassNames };
 export type CheckboxProps = ComponentProps<'input'> & {
   /** Whether this component should be unstyled. */
   unstyled?: undefined | boolean,
+
+  /** Whether the checkbox is in indeterminate state (minus sign) */
+  indeterminate?: undefined | boolean,
 };
 /**
- * Checkbox control.
+ * A simple Checkbox control, just the &lt;input type="checkbox"&gt; and nothing else..
```

Comment:
Just as a curiosity: if you enter HTML on the comment, Storybook renders it directly (i.e. a checkbox on the description), hence the `&lt;` and `&gt;`.